packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -423,7 +423,7 @@ class IOSDevice extends Device {
           deviceLogReader,
           portForwarder: portForwarder,
           throttleDuration: fallbackPollingDelay,
-          throttleTimeout: fallbackThrottleTimeout ?? const Duration(seconds: 5),
+          throttleTimeout: fallbackThrottleTimeout ?? const Duration(seconds: 20),
```

Comment:
If this increase is an experiment, how about making the results more unambiguous by making this timeout *much* bigger. Maybe 5-10 minutes?

Reply:
Done